main.py

```diff
@@ -199,11 +200,10 @@ def handle_syscall(syscall_id, syscall_object, entering, pid):
         (221, True): fcntl64_entry_handler,
         (196, True): lstat64_entry_handler,
         (268, True): statfs64_entry_handler,
-        (265, True): clock_gettime_entry_handler,
```

Comment:
This clock_gettime call is actually seperate from the timer stuff I was working on, and was already there when i started